h2o-jetty-9/src/main/java/water/webserver/jetty9/Jetty9Helper.java

```diff
@@ -41,7 +44,12 @@
     Server createJettyServer(String ip, int port) {
         System.setProperty("org.eclipse.jetty.server.Request.maxFormContentSize", Integer.toString(Integer.MAX_VALUE));
 
-        final Server jettyServer = new Server();
+        QueuedThreadPool pool = new QueuedThreadPool();
```

Comment:
Do we need to do the same thing for Jetty 8?

Reply:
I have manually tested Jetty 8 in context of Sparkling Water and it worked fine, the thread did not appear.

I can't also find this class in our Jetty 8 and from their git it seems like it wasn't put into jetty 8 https://github.com/eclipse/jetty.project/commit/fa412b4668e48590e89ad364541795fa73ba9ddc#diff-058fa4e5cdc53ae1c61956cddc416416

Reply:
I meant `QueuedThreadPool` - it is part of Jetty 8 codebase

Reply:
We do it, but just not in the constructor -> https://github.com/h2oai/h2o-3/pull/4110/files#diff-8fb7d504641963d01ccfd35dff8b56aeR55 Different jetty api

Reply:
oh ok - missed that file, thanks